packages/core/src/Badge/Badge.styles.tsx

```diff
@@ -13,7 +13,7 @@ export const { staticClasses, useClasses } = createClasses("HvBadge", {
   badgeContainer: { width: 0 },
   badgePosition: {},
   badge: {
-    borderRadius: theme.space.xs,
+    borderRadius: theme.space.sm,
```

Comment:
In one hand, shouldn't we be using one of our `radii` units (eg. `radii.full`)?
In the other hand, it won't matter either way (`sm`, `xs`, `full`, ...) because the badge's height is always `16px`